components/script/dom/htmltemplateelement.rs

```diff
@@ -32,9 +34,14 @@ impl HTMLTemplateElement {
         prefix: Option<Prefix>,
         document: &Document,
     ) -> HTMLTemplateElement {
+        let can_gc = CanGc::note();
         HTMLTemplateElement {
             htmlelement: HTMLElement::new_inherited(local_name, prefix, document),
-            contents: MutNullableDom::new(None),
+            contents: MutNullableDom::new(Some(
```

Comment:
It's already lazily-initialized in the `Content` method, so we don't need to initialize it here. And we can avoid creating a temporary `DocumentFragment` when using DSD.